intel/init.py

```diff
@@ -39,14 +39,83 @@ def init(conf_dir, num_exclusive_cores, num_shared_cores,
         logging.info("Could not read SST-BF label from the node metadata: {}"
                      .format(err))
 
+    sst_cp = False
```

Comment:
Please see my comment below first.

We may need some validation to ensure that SST-BF and SST-CP aren't requested at the same time.